v0.4.0/Azure-NDv4-A100-quickscan/dataset_description.md

```diff
@@ -0,0 +1,18 @@
+# Azure NDv4 Quick-scan Golden dataset on SuperBench v0.4
+
+The dataset is generated from 1939 NDv4 nodes' SuperBench v0.4 raw results.
+
+The results are the averaged data among the same type of metrics. For example, we aggregate IB loopback bandwidth from 8 NICs together and do the average, aggregate kernel launch time from 8 GPUs together and do the average， etc.
```

Comment:
did you "aggregate IB loopback bandwidth from 8 NICs"?

Reply:
"average，" -> "average,"
pls use half-width symbol instead of full-width

Reply:
ib_write_8388608_ib*_bw has been averaged to the same value

Reply:
it's not averaged? still ib_write_8388608_ib0_bw, ib_write_8388608_ib1_bw, etc.

Reply:
the values of them are the same, but I need to keep all full metric names. we didn't support the regex metric name in baseline currently.